src/app/github-link/github-link.component.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  Component,
+  Input,
+  ViewEncapsulation,
+  OnChanges,
+  SimpleChanges
+} from '@angular/core';
+
+import { 
+  HttpClient 
+} from "@angular/common/http";
+import { Observable } from "rxjs/Observable";
+
+/**
+ * GitHub Link component. This shows a formatted link to GitHub with an
+ * indicator of whether the GitHub issue is open or closed. Works only
+ * for public remote repositories for now.
+ */
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'github-link',
+  styles: [ `
+    .gh-link-hidden { display: none; }
+    .gh-link-open { color: red; }
+    .gh-link-closed { color: green; }
+    .gh-link-error { color: orange; }
```

Comment:
I believe we would want to use the PatternFly colors here; @color-pf-green, @color-pf-red, @color-pf-orange, etc. These colors map to @color-pf-green-400, @color-pf-red-100, and @color-pf-orange-400.